pkg/virt-handler/vm_test.go

```diff
@@ -2409,7 +2409,7 @@ var _ = Describe("VirtualMachineInstance", func() {
 				},
 			}
 
-			condition, isBlockMigration := controller.calculateLiveMigrationCondition(vmi)
+			condition, isBlockMigration := controller.calculateLiveMigrationCondition(vmi, nil)
```

Comment:
Can you also add a unit test where you call `calculateLiveMigrationCondition` with `condManager *controller.VirtualMachineInstanceConditionManager != nil` ? For example just an Idempotence sanity check.